service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -376,8 +377,12 @@ private Location getLocationByOrderIdThroughLazyInitialization(Order order) {
     public PersonalDataDto getSecondPageData(String uuid) {
         User currentUser = userRepository.findByUuid(uuid);
         List<UBSuser> ubsUser = ubsUserRepository.findUBSuserByUser(currentUser);
+
         if (ubsUser.isEmpty()) {
-            ubsUser.add(UBSuser.builder().id(null).build());
+            UBSuser ubSuser = UBSuser.builder().id(null).build();
+            List<UBSuser> mutableUbsUserList = new ArrayList<>(ubsUser);
+            mutableUbsUserList.add(ubSuser);
```

Comment:
could you please explain, why we should to create new list?

Reply:
When i covered all lines with tests, previous format had errors.